client/src/components/modals/DetailsModal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Modal, Header } from 'semantic-ui-react';
+import 'semantic-ui-css/semantic.min.css';
+
+function DetailsModal({ session }) {
+  const [open, setOpen] = React.useState(false);
+  return (
+    <Modal
+      className="modal-container"
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={<Button className="join-button">JOIN</Button>}
+    >
+      <Modal.Content>
+        <Modal.Description>
+          <Header>Join {session.creator}</Header>
+          <p id="form-text">
+            Class: {session.class}
+            <br />
+            Location: {session.location}
+            <br />
+            Expected Group Size: {session.attendees}
+            <br />
+            Notes from {session.creator}
+            <div className="notes-bg">
+              <p>{session.notes}</p>
+            </div>
```

Comment:
Can you add the startTime to this too? Make sure to convert it from [epoch time](https://www.epochconverter.com/) to a Date